### PR TITLE
Add logio container for qa and api logs.

### LINKED ISSUE #13: Shows logs in web page
Mostrar los logs de docker en alguna página web, así los podemos ver todos de manera fácil.

### PR DESCRIPTION
**Pull request by pgrill**
Resolves #13

### CODE REVIEW

File: docker-compose.prod.yml [hunk 1]
```diff
@@ -46,6 +48,14 @@ services:
...
       - traefik.docker.network=proxy
       - traefik.http.routers.api.rule=Host(`${DOMAIN}`) && Path(`/api`)
+      - logio=yes
+
+  logio:
+    image: geniousphp/logio
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock:ro
+    ports:
+      - 28778:28778
```
Comment (bryant1410) [inline docker-compose.prod.yml]: Can't we expose another port for this, maybe HTTP(S) through Traefik?

File: docker-compose.prod.yml [hunk 2]
```diff
@@ -16,6 +16,8 @@ services:
       - ANSWER_THREADS=8
       - BATCH_SIZE=672
       - FLASK_ENV=production
+    labels:
+      - logio=yes
```
Comment (bryant1410) [inline docker-compose.prod.yml:20]: Maybe we also need logs from another container like elastic's? Or you think it isn't necessary?

**Review by bryant1410 — APPROVED** [review]

File: docker-compose.prod.yml (hunk 2 above)
Comment (pgrill) [reply docker-compose.prod.yml:20]: Yes, we can add all the containers that we want. In my local tests, the elastic's log didn't give me relevant information.

Comment (bryant1410) [reply docker-compose.prod.yml:20]: Okay. Then it's fine.

File: docker-compose.prod.yml (hunk 1 above)
Comment (pgrill) [reply docker-compose.prod.yml]: The issue with exposing it behind Traefik is that we need to create a subdomain for it.
If we append a path (like we did with django), the logio container didn't upload correctly (something similar what happens with the django admin).  Maybe I am configuring wrong Traefik 🤔 

For meanwhile, I think that we can just expose the port in the firewall and later with the domain create a subdomain for it.

Comment (bryant1410) [reply docker-compose.prod.yml]: Sounds good. Can you add a comment?

Comment (pgrill) [reply docker-compose.prod.yml]: Sure. I will add it.

**Comment by bryant1410** [comment]
Feel free to merge!

btw, I see checks are running twice. We need to fix this at some point.

**Pull request merged**